src/game/shared/tf/tf_weapon_passtime_gun.cpp

```diff
@@ -552,15 +538,12 @@ void CPasstimeGun::ItemPostFrame()
 				pOwner->m_Shared.SetPasstimePassTarget( pNewTarget );
 				pCurrentTarget = pNewTarget;
 
+#ifdef CLIENT_DLL
 				// play the lock-on sound for the player
-				CRecipientFilter filter;
-				filter.AddRecipient( pOwner );
-				EmitSound( filter, pOwner->entindex(), kTargetHightlightSound );
-
-				// now play it for the target
-				filter.RemoveAllRecipients();
-				filter.AddRecipient( pCurrentTarget );
-				EmitSound( filter, pCurrentTarget->entindex(), kTargetHightlightSound );
+				pOwner->EmitSound( kTargetHightlightSound );
+				// play it for the target
+				pCurrentTarget->EmitSound( kTargetHightlightSound );
```

Comment:
The original code was transcribed incorrectly here when swapping logical sides -- it doesn't make sense to play this sound on the carrier's client. This means that no sound is heard when another player is locking on to you.